### DISCUSSION TITLE
Update on version 2.0

### DISCUSSION THREAD

**Discussion by sirajahmadzai** [jfree/jfreechart#419] [General]
Hello, 

I am currently using JFreeChart and was wondering if there is an estimated release date for version 2.0. I understand that development timelines can change, but any information or updates you could provide would be greatly appreciated.

**Comment by synthfi** [comment]
I am also interested in this, since 2.0 appears to be modular

**Reply by trashgod** [reply]
While not released, v2.0.0 can be built and used locally, as suggested [here](https://github.com/jfree/jfreechart/issues/219); a custom approach is examined [here](https://github.com/jfree/jfreechart/discussions/374).

**Reply by synthfi** [reply]
yes that is an option. I would prefer to be able to download through maven and not have to do any specific configuration on my end though
